js/gallery.js

```diff
@@ -3,19 +3,18 @@ import { showPicture } from './big-picture.js';
 
 const container = document.querySelector('.pictures');
 
-// Обработчик кликов
-const renderGallery = (pictures) => { // Принимает массив с данными
-  container.addEventListener('click', (evt) => { // Отслеживаем события в контейнере, в evt подставляется объект события
-    const thumbnail = evt.target.closest('[data-thumbnail-id]'); // Проверка у элемента наличие дата атрибута, closest ищет ближайший родительский элемент
+const renderGallery = (pictures) => {
+  container.addEventListener('click', (evt) => {
+    const thumbnail = evt.target.closest('[data-thumbnail-id]');
 
     if (! thumbnail) {
```

Comment:
Мне кажется такое сложно читать
```suggestion
    if (!thumbnail) {
```